Scemantic Focus

```diff
@@ -0,0 +1,110 @@
+---
+id: 
+name: Scemantic Focus 
+rule_type: atomic
+
+description: | 
+ This rule checks that every element part of the sequential focus navigation has a semantic role.
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+
+authors:
+- Carlos Duarte
+- Damien Engels
+- Marie Trudelle 
+- Ramit Garg
+---
+
+## Applicability
+
+The rule applies to all element in the sequential focus navigation excepts scrollable content element.
+
+## Expectation
+
+Target element has a semantic role.
+
+## Assumptions
+
+- Giving focus to scrollable content elements is the only way to scroll non interactive content using a keyboard.
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- https://www.w3.org/TR/WCAG20-TECHS/G202.html
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+Focusable element have semantic role 
+
+```html
+<a href="xyz">Go to order</a>
+```
+
+#### Passed Example 2
+
+Focusable element button have semantic role 
+
+```html
+<button>click over here</button>
+```
+
+#### Passed Example 3
+
+Element with tabindex 0 has explicit semantic role 
+
+```html
+<div id="id1" role="button" tabindex="0">Go</div>
+```
+
+#### Passed Example 4
+
+
+### Failed
```

Comment:
Add a test where you override the role of an element that's focusable by default.